apps/src/templates/GameButtons.jsx

```diff
@@ -19,12 +21,28 @@ export const FinishButton = () => (
   </button>
 );
 
+const sendAnalyticsEvent = () => {
+  const isSignedOut =
+    document.querySelector('script[data-issignedout="true"]') !== null;
```

Comment:
Curious why we're using the script data attribute instead of reading from `currentUserRedux`? I just want to make sure we don't have multiple sources of truth as to whether a user is signed in...